intel_extension_for_transformers/neural_chat/models/model_utils.py

```diff
@@ -394,13 +394,38 @@ def load_model(
              or re.search("codellama", model_name, re.IGNORECASE)
             ) and ipex_int8
         ):
-            with smart_context_manager(use_deepspeed=use_deepspeed):
-                import intel_extension_for_pytorch
-                from optimum.intel.generation.modeling import TSModelForCausalLM
-                model = TSModelForCausalLM.from_pretrained(
-                        model_name,
-                        file_name="best_model.pt",
-                 )
+        with smart_context_manager(use_deepspeed=use_deepspeed):
+            import intel_extension_for_pytorch
```

Comment:
Shall we add a checker like is_ipex_available? We should limit the hard depdencies.

Reply:
yes, fixed.